src/data/queries/bank.ts

```diff
@@ -38,8 +42,13 @@ export const useInitialTokenBalance = () => {
             denom: token,
             chain: chainID,
           }
-        })
-      )) as CoinBalance[]
+        }),
+        ...native.map(async ({ token, chains }) => ({
```

Comment:
Don't put this one here, pull the latest changes from the beta branch and do it in InitBankBalances